salesagility/SuiteCRM
# V8 API swagger.json

## Description



Created a route for the V8 API swagger (Technically a missing feature / bug fix as this was available in the v8 API).

## Motivation and Context

Noticed the [V8 API docs](https://docs.suitecrm.com/developer/api/developer-setup-guide/getting-available-resources/) still point to old v8 api swagger.json which is likely to cause confusion.

## How To Test This

1. Setup the V8-API using the documentation.
2. Attempt a GET request to ```{{suitecrm.url}}/V8/meta/swagger.json```.
3. Check the response returns the valid V8 API swagger schema.

## Types of changes


### Final checklist
